composer.json

```diff
@@ -169,7 +173,8 @@
       "/src/Features",
       ".gitignore",
       ".travis.*",
-      "ci/qa/behat.yml"
+      "ci/qa/behat.yml",
+      "/vendor/google/apiclient-services/src/*"
```

Comment:
Are you intentionally excluding this folder? I would think that code is important to make it work.

Reply:
Yes, this is intentional. /vendor/google/apiclient-services/src/* contains automatically generated PHP classes for ALL the Google APIs. We do not use these. The classes required for the apiclient are not removed, these are in different directories.
There two issues with this:
1. It is huge
2. It contains filenames that are so long that it breaks the MAX_PATH length, which makes the tar step file.